classes/mcPy/McPy.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import multiprocessing
+import os
+import sys
+
+from .Parser import Parser
+import classes.Server as Server
+
+
+def get_available_core():
+    try:
+        avail_cores = len(os.sched_getaffinity(0))
+    except AttributeError:
+        # Fix for windows, which doesnt support getaffinity
+        logging.warning(
+            "Falling back to multiprocessing cpu_count to calc cores. Most likely getaffinity is not supported on "
+            "your OS")
+        avail_cores = multiprocessing.cpu_count()
+    return avail_cores
+
+
+def _launch(parser: Parser):
+    try:
+        logging.info("The Blackfire probe")
```

Comment:
This makes little to no sense as it's repeated again if it's not installed (makes a bit more sense if it is installed though)

Reply:
Nice catch :D

Reply:
Done